docs/ocis/development/testing.md

```diff
@@ -37,83 +37,112 @@ You can invoke two types of test suite runs:
 
 ### Run full test suite
 
-The names of the full test suite make targets have the same naming as in the CI pipeline. The available local ocis specific test suites are `apiAccountsHashDifficulty`, `apiArchiver`, `apiContract`, `apiGraph`, `apiSpaces`, `apiSpacesShares`, `apiCors`, `apiAsyncUpload`. They can be run with `ocis` storage and `S3` storage.
+#### Local oCIS tests (prefix `api`)
```

Comment:
```suggestion
#### Local oCIS Tests (prefix `api`)
```
Pls use [headlinecapitalisation.com](https://headlinecapitalization.com) for proper writing of section headlines.